addons/respawn/functions/fnc_updateRallypoint.sqf

```diff
@@ -17,7 +17,8 @@
  * Public: No
  */
 
-params ["_rallypoint", "_side", "_position"];
+params ["_rallypoint", "_side"];
+private _position = param [2, getpos _rallypoint];
```

Comment:
why not:
```suggestion
params ["_rallypoint", "_side", ["_position", getPos _rallypoint]];
```

Reply:
`_rallypoint` is undefined when making the array (the array is fully constructed and then passed to `params`)